usr/share/rear/conf/default.conf

```diff
@@ -3247,6 +3247,47 @@ VEEAM_BACKUPID="${VEEAM_BACKUPID:-}"
 # End of BACKUP=VEEAM default settings.
 ####
 
+# BACKUP=COVE (Cove Data Protection; Backup Manager)
+##
+# ReaR support for Cove Data Protection (Cove).
+# The external software component Cove is supported and maintained by N-able.
+
+REQUIRED_PROGS_COVE=(
+    base64
+    chgrp
+    dirname
+    ln
+    sleep
+    /bin/kill
+)
+
+# Let the Backup Manager know that it is installing or running inside the rescue environment.
+KERNEL_CMDLINE_COVE="cove_rescue_media"
+
+# Cove install directory.
+# The default value is configured in site.conf during the Backup Manager installation.
+COVE_INSTALL_DIR=
+
+# The URL used to download the Backup Manager installation package.
+# The default value is configured in site.conf by the Backup Manager.
+# It can be overridden via the `cove_installer` boot parameter.
+# If left empty, the user will be prompted at recovery time to provide the URL.
+COVE_INSTALLER_URL=
```

Comment:
So far all ReaR integrations are built so that the resulting rescue image doesn't have to download software to function. Is there a special reason to dynamically install the Backup Manager during recovery? I'd be worried that this reduces the reliability of a ReaR / COVE rescue image as this software installation step could fail during recovery. That is the reason why we prefer building a "ready to use" rescue image that contains all required software components and configuration.